canvas_modules/common-canvas/src/tooltip/tooltip.scss

```diff
@@ -34,6 +34,17 @@
 	word-wrap: break-word;
 	max-width: 228px;
 	border-radius: 2px;
+	.bx--link {
```

Comment:
Why do we have to override the link coloring?

Reply:
Default color for `bx--link` class is `$link-01`
![Screen Shot 2022-04-11 at 1 20 10 PM](https://user-images.githubusercontent.com/25124000/162825216-2e264928-cfee-4ea1-8115-b315c108b7fd.png)

Overriding this color with `$inverse-link`. Copied this CSS  - 
![Screen Shot 2022-04-11 at 1 21 53 PM](https://user-images.githubusercontent.com/25124000/162825341-056cf0ed-484f-4532-81f5-feb4f5e45626.png)

Carbon adds color override in tooltip - https://v10.carbondesignsystem.com/components/tooltip/code/
![Screen Shot 2022-04-11 at 1 23 35 PM](https://user-images.githubusercontent.com/25124000/162825714-c55e40d4-c48a-4391-8238-a9b4bbe0c446.png)